tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/manifest.ini

```diff
@@ -30,7 +30,16 @@ online = true
 smoketest = true
 
 [test_settings_cell_data.py]
-skip-if = device == "desktop"
+skip-if = device == "desktop" || device == "flame"
+carrier = true
+smoketest = true
+sanity = true
+
+[test_settings_dsds_cell_data.py]
+# This test doesn't require 2 SIMs to be executed, but requires a DSDS-compatible device
```

Comment:
You need `dsds = true` to make it run on devices with 2 sims. I made the comment clearer.